Phoenix/Presence.cs

```diff
@@ -1,3 +1,9 @@
+using UnityEngine;
```

Comment:
I don't think we should import Unity in the main project, as it is meant to work with Unity as well as any C# project.